community/server/src/test/java/org/neo4j/server/rest/transactional/Neo4jJsonCodecTest.java

```diff
@@ -218,4 +221,31 @@ public void shouldWriteAMapContainingNullAsKeysAndValues() throws IOException
         // then
         verify( jsonGenerator, times( 1 ) ).writeFieldName( "null" );
     }
+
+    @Test
+    public void testGeographicPointWriting() throws IOException
+    {
+        //Given
+        GeographicPoint value = new GeographicPoint( 12.3, 45.6, CRS.WGS84() );
```

Comment:
The `Neo4jJsonCodec` need to know how to write `graphdb.spatial.Point` not an internal scala type